wct-legacy.conf.json

```diff
@@ -0,0 +1,28 @@
+{
+  "npm": true,
+  "moduleResolution": "node",
+  "suites": [
+    "./test-legacy.html"
+  ],
+  "clientOptions": {
+    "mochaOptions": {
+      "timeout": 30000
+    }
+  },
+  "plugins": {
+    "sauce": {
+      "browsers": [
+        {
+          "browserName": "internet explorer",
+          "platform": "Windows 10",
+          "version": "11.285"
+        },
+        {
+          "browserName": "safari",
```

Comment:
Does Safari 11.1 need the legacy build?

Reply:
Probably not. I'll move it up.

Now that I think about it, it's probably valuable to ensure latest Chrome works against the legacy bundle, as the code is changed very significantly.

Reply:
Done